demo/website/src/components/Project.js

```diff
@@ -1,25 +1,28 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import './Project.css';
+
 class Project extends Component {
   render() {
     return (
       <li className="ecl-list-item">
-        <a href="" className="ecl-link ecl-list-item__link">
+        <a
+          href={this.props.project._source.project_website}
+          target="_blank"
+          className="ecl-link ecl-list-item__link"
+        >
           <div className="ecl-u-sr-only">List item</div>
 
           <div className="ecl-list-item__body">
             <div className="ecl-list-item__meta">
               <div className="ecl-meta">
-                {this.props.project._source.related_links
-                  ? this.props.project._source.related_links.map(
-                      (related, index) => (
-                        <span className="ecl-meta__item" key={index}>
-                          {related.label}
-                        </span>
-                      )
-                    )
-                  : ''}
+                {(this.props.project._source.related_links || []
```

Comment:
I'm not sure the change makes it more readable to be honest, but ok

Reply:
It's a safety net. Alternatives, opinions?

Reply:
My suggestion was in the previous version which checks presence and if non-present, just not outputting elements.